templates/admin/usergroup/list.html.twig

```diff
@@ -0,0 +1,125 @@
+{#
+Twig template to list users.
+
+@TODO Figure out how to docblock twig templates
+
+@TODO The usergroup_admin_php script has not yet been converted to twig/bootstrap
+(full) and because of that we're doing a dynamic load of that content into a
+bootstrap modal. Eventually this could be abstracted into a twig snippet RD 2017-06-03
+
+@package OpenEMR
+@subpackage Usergroup
+@author Robert Down <robertdown@live.com>
+@copyright Copyright (c) 2017 Robert Down
+@license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+{% extends "global/snippet/navbar.html.twig" %}
+
+{% block header %}
+{{ setupHeader('common') }}
+<script type="text/javascript">
+    $(document).ready(function(){
+        /**
+         * Use the data-content element to load remote content into a modal.
+         */
+        $(".modal").on("show.bs.modal", function(e) {
+            var modal = $(this);
+            var triggeringElement = $(e.relatedTarget);
+            var content = triggeringElement.data("content");
+            var body = modal.find(".modal-body");
+            body.load(content);
+        });
+
+        $("#inactive").on("change", function(e) {
+            top.restoreSession();
+            $(this).parents("form").submit();
+        });
+    });
+
+    function authorized_clicked() {
+        var f = document.forms[0];
+        f.calendar.disabled = !f.authorized.checked;
+        f.calendar.checked  =  f.authorized.checked;
+    }
+
+    {% if alertmsg %}
+    alert({{ alertmsg }});
```

Comment:
javascript literals get different escaping. in php, use addslashes, not sure which to use in twig.